demos/time_series_demo/python/models.lst

```diff
@@ -0,0 +1 @@
+time-series-forecasting-electricity-0001
```

Comment:
```suggestion
# TODO: time-series-forecasting-electricity-0001
```

(We'll uncomment it after the model is published.)

Reply:
Even better
```suggestion
# TODO: time-series-forecasting-electricity-????
```
Diff breaks on outdated comments :(

Reply:
:ok_hand: